rllib/core/models/catalog.py

```diff
@@ -25,93 +25,6 @@
 from ray.rllib.utils.spaces.space_utils import get_base_struct_from_space
 
 
-def _multi_action_dist_partial_helper(
```

Comment:
Moved down for making the main Catalog class in this file more prominent. We should generally always move private functions to the end of files to avoid confusion and make the main class(es) in a file more visible.